### PR TITLE
Add Paul's Hawaii recommendations

### LINKED ISSUE #10: Create a file for Hawaii recommendations
# Hawaii

- [ ] Create a branch
- [ ] Add a commit
- [ ] Open a pull request
- [ ] Have a discussion
- [ ] Seek approval
- [ ] Merge pull request

### PR DESCRIPTION
**Pull request by ptsushima**
close #10

### CODE REVIEW

File: ptsushima-hawaii.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Hawaii
+
+### Places to eat
+- Grace's
+- Rainbows
+
```
Comment (Dscholz23) [inline ptsushima-hawaii.md]: Can I get Hawaiian Ice at one of these places?

**Comment by Dscholz23** [comment]
🐉

**Comment by arde5730** [comment]
More stars. Good good.

**Review by ptsushima — COMMENTED** [review]
Nice beaches! 🏖

**Pull request merged**